tools/migrate_capture_one_pro.sql

```diff
@@ -0,0 +1,46 @@
+ATTACH 'data.db' AS data;
```

Comment:
Can you add a set of comments at the start of the script about the setup before starting the script. From where it should be run? What are the db to copy before starting... etc. A bit of context for first time users.

And also what it does. As I understand it, it creates a tag for each Capture One collection? All this must be clearly described. Thanks.

Reply:
I've added a section at the start. Let me know if that's sufficient.